src/Microsoft.Azure.WebJobs.Extensions.EventHubs/Config/EventHubExtensionConfigProvider.cs

```diff
@@ -161,7 +161,21 @@ private static EventData ConvertBytes2EventData(byte[] bytes)
 
                     if (!string.IsNullOrEmpty(partitionKey))
                     {
-                        byte[] body = o.GetValue("Body", StringComparison.OrdinalIgnoreCase)?.ToObject<byte[]>();
+                        byte[] body;

```

Comment:
There is a test for byte[]. Can we also add tests for JObject and string?

Reply:
I removed the String check, I did not find a way to distinguish between byte[] and string. https://stackoverflow.com/questions/36285334/distinguish-byte-and-string-from-json-net